README.md

```diff
@@ -58,12 +58,36 @@ Of those, this plugin currently uses:
 
 In addition to the above standardised tags, this plugin also adds:
 
+  - `component` (sent to Zipkin as "lc", for "local component")
   - `kong.api` (deprecated)
   - `kong.consumer`
   - `kong.credential`
   - `kong.node.id`
   - `kong.route`
   - `kong.service`
-  - `kong.balancer.try`
-  - `kong.balancer.state`: see [here](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/balancer.md#get_last_failure) for possible values
-  - `kong.balancer.code`
+  - `kong.try_[n].ipv4`
```

Comment:
all of this should become a child span with kind=CLIENT

assuming there is always at least one http "try", each of these subspans should be a CLIENT span and the parent span should not. The encoding done here will make search unusable, but that's not the only reason subspans should be used. This style also breaks dependency graph linking

Reply:
I did as requested. Notice that previously the load balancer spans were children of the *proxy* span, which was of kind=CLIENT. I have made them children of the *request* span now, which is the only span of kind "not CLIENT" (it is a SERVER span).